src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -2,13 +2,25 @@
 
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.validator.Author;
+import mate.academy.validator.CoverImage;
+import mate.academy.validator.Description;
+import mate.academy.validator.Isbn;
+import mate.academy.validator.Price;
+import mate.academy.validator.Title;
 
 @Data
 public class CreateBookRequestDto {
+    @Title
```

Comment:
You can use instead of creating your own validation annotation
```suggestion
    @NotBlank(message = "Title must not be null or empty")
```